### ISSUE TITLE
Module 4: Fields blank while in loop

### ISSUE THREAD

**Issue by meekk1** [episphere/quest#140]
[HOMEADD2] [IF RESPONDENT LEAVES ALL FIELDS BLANK IN HOMEADD1, PROMPT WITH BELOW. IF RESPONDENT LEAVES CITY, STATE, ZIP, OR COUNTRY BLANK IN HOMEADD1, PROMPT WITH BELOW.]
**AND**
[HOMEADD3] [IF RESPONDENT LEAVES STREET NUMBER AND STREET NAME BLANK IN HOMEADD1 BUT COMPLETED THE CITY, STATE, ZIP, AND/OR COUNTRY FIELDS IN HOMEADD1 AND/OR HOMEADD2, PROMPT WITH BELOW. IF ALL FIELDS LEFT BLANK IN HOMEADD1 AND HOMEADD2, SKIP TO HOMEADD4.]

**Comment by danielruss** [comment]
@ashreves I'm confused.  If the Respondent leaves either city, state, or country blank in HOMEADD, you want HOMEADD2 shown. This contradicts ALL FIELDS blanks.

@ruggz13 we may need an element that becomes a text field if a value is blank, or it becomes either a locked text field or a span.  See the question in module 4.

**Comment by ashreves** [comment]
@danielruss If participant leaves all of HOMEADD blank, they should see HOMEADD2.

If a participant leaves city, state, zip, OR country blank, they should see HOMEADD2.

For example, if a participant ONLY fills out their street name in HOMEADD, they should get HOMEADD2. But if a participant fills out everything but their street name in HOMEADD, they SHOULD NOT get HOMEADD2.

**Comment by ashreves** [comment]
It's similar to the SSN follow-up questions. Instead of asking them to fill out the whole field again, we only ask for a certain portion of the fields.

**Comment by danielruss** [comment]
Thank you Alaina

**Comment by meekk1** [comment]
This is what I have so far for the first address loop in Module 4. Please correct me if I'm wrong. And I'm not sure how to list the addresses in HOMEADDSUM.

https://gist.github.com/meekk1/0ce5b337f88e55f3db36ef8052b24b00

**Comment by ashreves** [comment]
Looking back at NORC's documentation, I found this note that might be relevant. I am not sure if this has been implemented yet or not. 

"HOMEADD1, SEASADD1, CHILDADD1, CURWORK1, PREWORK1, CURSCH1 | NCI requested that there be a soft prompt when respondents enter addresses to prompt respondents to enter a piece of the address if the street number, street name, city, state/province, zip, or country was missing. A request response was not wanted if only the apartment number was missing. NORC did not resolve this issue as it would require programming unique to Qualtrics."
